python/ptest/cases/gomspace_checkout_case.py

```diff
@@ -133,29 +133,36 @@ def write_state(self, string_state, state_value):
         if ppt_mode_cmd == ppt_mode_updated:
             print("Could not update pptmode")
 
-        heater_cmd = self.str_to_bool(read_state(self, "gomspace.heater_cmd"))
```

Comment:
Instead of a custom str_to_bool function you could use json.loads? Up to you

Reply:
Honestly I think it's worth modifying StateSession to automatically handle bools, since it makes all other code cleaner.